src/status_im/ui/screens/wallet/main/views.cljs

```diff
@@ -67,10 +67,12 @@
     [react/view {:style wallet.styles/value-variation}
      [react/text {:style wallet.styles/value-variation-title}
       (i18n/label :t/wallet-total-value)]
-     [change-display change]]
+     [react/view {:style (if (pos? change) wallet.styles/today-variation-container-positive wallet.styles/today-variation-container-negative)}
```

Comment:
Why the change here?

Reply:
This probably came from `develop`, will revert it.